awx/ui/src/components/Schedule/shared/ScheduleForm.test.js

```diff
@@ -1096,40 +1233,64 @@ describe('<ScheduleForm />', () => {
       expect(wrapper.find('FormGroup[label="Run on"]').length).toBe(0);
 
       nonRRuleValuesMatch();
-      expect(wrapper.find('select#schedule-frequency').prop('value')).toBe(
-        'week'
-      );
-      expect(wrapper.find('input#schedule-run-every').prop('value')).toBe(1);
-      expect(wrapper.find('input#end-never').prop('checked')).toBe(false);
-      expect(wrapper.find('input#end-after').prop('checked')).toBe(false);
-      expect(wrapper.find('input#end-on-date').prop('checked')).toBe(true);
       expect(
-        wrapper.find('input#schedule-days-of-week-sun').prop('checked')
+        wrapper.find('FrequencySelect#schedule-frequency').prop('value')
+      ).toEqual(['week']);
+      expect(
+        wrapper
+          .find('input#schedule-run-every-frequencyOptions-week')
+          .prop('value')
+      ).toBe(1);
+      expect(
+        wrapper.find('input#end-never-frequencyOptions-week').prop('checked')
+      ).toBe(false);
+      expect(
+        wrapper.find('input#end-after-frequencyOptions-week').prop('checked')
       ).toBe(false);
       expect(
-        wrapper.find('input#schedule-days-of-week-mon').prop('checked')
+        wrapper.find('input#end-on-date-frequencyOptions-week').prop('checked')
       ).toBe(true);
       expect(
-        wrapper.find('input#schedule-days-of-week-tue').prop('checked')
+        wrapper
+          .find('input#schedule-days-of-week-sun-frequencyOptions-week')
+          .prop('checked')
       ).toBe(false);
       expect(
-        wrapper.find('input#schedule-days-of-week-wed').prop('checked')
+        wrapper
+          .find('input#schedule-days-of-week-mon-frequencyOptions-week')
+          .prop('checked')
       ).toBe(true);
       expect(
-        wrapper.find('input#schedule-days-of-week-thu').prop('checked')
+        wrapper
+          .find('input#schedule-days-of-week-tue-frequencyOptions-week')
+          .prop('checked')
       ).toBe(false);
       expect(
-        wrapper.find('input#schedule-days-of-week-fri').prop('checked')
+        wrapper
+          .find('input#schedule-days-of-week-wed-frequencyOptions-week')
+          .prop('checked')
+      ).toBe(true);
+      expect(
+        wrapper
+          .find('input#schedule-days-of-week-thu-frequencyOptions-week')
+          .prop('checked')
+      ).toBe(false);
+      expect(
+        wrapper
+          .find('input#schedule-days-of-week-fri-frequencyOptions-week')
+          .prop('checked')
       ).toBe(true);
       expect(
-        wrapper.find('input#schedule-days-of-week-sat').prop('checked')
+        wrapper
+          .find('input#schedule-days-of-week-sat-frequencyOptions-week')
+          .prop('checked')
       ).toBe(false);
       expect(
         wrapper.find('DatePicker[aria-label="End date"]').prop('value')
       ).toBe('2021-01-01');
       expect(
         wrapper.find('TimePicker[aria-label="End time"]').prop('value')
-      ).toBe('1:00 AM');
+      ).toBe('12:00 AM');
```

Comment:
I think this was a timezone/DST bug in the previous version...? America/New_York should be UTC-5:00 in January